projects/plugins/boost/app/assets/src/js/utils/create-hash-source.ts

```diff
@@ -0,0 +1,45 @@
+/**
+ * External dependencies
+ */
+import { createHashHistory } from 'history';
+
+function createHashSource() {
```

Comment:
I don't understand what this is function is for - can we please have a docblock comment describing its purpose?

Reply:
Added